common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -135,8 +135,70 @@ class LibraryContentModule(LibraryContentFields, XModule, StudioEditableModule):
     any particular student.
     """
 
+    def __init__(self, *args, **kwargs):
+        self.children = []
+        self._selected_set = None
+        super(LibraryContentModule, self).__init__(*args, **kwargs)
```

Comment:
These might belong higher up (on XModule maybe?) but setting an empty value here for `self.children` and `self._selected_set` eliminates the need for a number of pylint disables.